README.md

```diff
@@ -51,3 +53,30 @@ npm run test
 ```plain
 npm run doc
 ```
+## Usage
+After installing *As*, you can import the object classes and functions that you need in your AssemblyScript file.
+
+For example, to use the `combination` function to determine the combination of 2 numbers, you can import and use it like this:
+```typescript
+import { combination } from  '@massalabs/as-proba';
+import { Args } from  '@massalabs/as-types';
+
+// This main function is called automatically when the smart contract is executed by the blockchain.
```

Comment:
I'm not sure it's still true

Reply:
I think now it is the constructor that is called automatically, so I am not sure if this statement is still true. I may be wrong

Reply:
oh yes